CRIS_CLDDET.NL

```diff
@@ -0,0 +1,53 @@
+&Cloud_Detect_Coeffs
+N__Band_Size(1:4)= 137, 123, 76, 12
+N__Bands(1:137,1)=
+    1,    5,    9,   13,   17,   18,   19,   20,   21,   22,
+   23,   24,   25,   26,   27,   28,   29,   30,   31,   32,
+   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
+   43,   44,   45,   46,   47,   48,   49,   50,   51,   52,
+   53,   54,   55,   56,   57,   58,   59,   60,   61,   62,
+   63,   64,   65,   66,   67,   68,   69,   70,   71,   72,
+   73,   74,   75,   76,   77,   78,   79,   80,   81,   82,
+   83,   84,   85,   86,   87,   88,   91,   92,   93,   94,
+   95,   96,   97,   99,  101,  105,  107,  109,  111,  113,
+  115,  116,  117,  118,  119,  120,  121,  122,  123,  124,
+  125,  133,  135,  137,  139,  141,  144,  147,  161,  173,
+  177,  181,  185,  195,  210,  221,  225,  229,  249,  257,
+  269,  273,  293,  301,  317,  333,  349,  369,  409,  433,
+  457,  481,  501,  549,  701,  705,  709
+N__Bands(1:123,2)=
+    3,    6,    7,    8,   10,   12,   14,   15,   16,   89,
+   90,  102,  103,  104,  106,  108,  110,  114,  126,  127,
+  129,  132,  134,  138,  140,  143,  145,  146,  148,  149,
+  150,  151,  153,  155,  156,  157,  158,  159,  162,  163,
+  164,  165,  166,  169,  170,  171,  172,  175,  180,  189,
+  200,  201,  205,  206,  214,  217,  218,  226,  228,  230,
+  231,  233,  236,  237,  240,  241,  245,  248,  252,  264,
+  265,  281,  285,  297,  324,  327,  361,  378,  389,  392,
+  400,  473,  493,  500,  503,  511,  527,  528,  529,  530,
+  531,  534,  538,  542,  544,  545,  547,  550,  553,  555,
+  590,  594,  598,  602,  606,  610,  614,  618,  622,  626,
+  645,  649,  653,  657,  661,  665,  685,  702,  703,  704,
+  706,  707,  713
+N__Bands(1:76,3)=
+  717,  725,  728,  729,  730,  731,  732,  733,  734,  735,
+  736,  741,  749,  757,  765,  773,  781,  789,  794,  797,
+  805,  806,  815,  822,  829,  839,  845,  853,  861,  868,
+  869,  872,  877,  885,  887,  893,  898,  900,  909,  912,
+  915,  917,  921,  929,  933,  941,  949,  957,  963,  965,
+  973,  975,  978,  981,  989,  991,  993,  996, 1005, 1014,
+ 1025, 1029, 1037, 1042, 1053, 1061, 1073, 1077, 1085, 1093,
+ 1101, 1109, 1117, 1125, 1133, 1141
+N__Bands(1:12,4)=
+ 1149, 1157, 1164, 1165, 1173, 1181, 1189, 1197, 1205, 1213,
+ 1221, 1251
+R__BT_Threshold(1)=0.60
+N__Window_Width(1)=6
+L__Do_Imager_Cloud_Detection = .TRUE.
+N__Num_Imager_Chans = 2
+N__Num_Imager_Clusters = 7
+N__Imager_Chans(:) = 15, 16
+R__Stddev_Threshold(:) = 0.4, 0.4
```

Comment:
All of your other namelist arrays are explicitly indexed.  Should these be as well?  i.e.:
```
N__Imager_Chans(1:2) = 15, 16
R__Stddev_Threshold(1:2) = 0.4, 0.4
```